### PR TITLE
column options styling modified

### PR DESCRIPTION
**Pull request by priyalpepcus**

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -10,6 +10,8 @@
     -->
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json">
     <link rel="icon" type="image/png" sizes="132x132" href="LOGO.png">
+    <link rel="stylesheet"
```
Comment (pepcus-pratik) [inline public/index.html]: Use the 'Font-awesome React' npm package instead.

File: src/components/AdminPanel.jsx [hunk 2]
```diff
@@ -217,15 +214,27 @@ class AdminPanel extends Component {
...
-              buttonText={'Student Information'}
-              onClick={this._setRedirectValue}
-            />
-          </div>
+        <div className="student-information-section">
+          <div className="student-registration-wrapper">
+            <div className="student-information-wrapper">
+              <div className = "student-information-header">Student Information</div>
+              <div className="student-information-text">
+                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit,
```
Comment (pepcus-pratik) [inline src/components/AdminPanel.jsx]: I believe we are updating this text in the next PR

File: src/components/AdvanceSearch.js [hunk 3]
```diff
@@ -85,6 +84,25 @@ class AdvanceSearch extends Component {
             <label htmlFor="deep_search">Deep Search</label>
           </div>
         </div>
+       {/* <label htmlFor = "search_input">
```
Comment (pepcus-pratik) [inline src/components/AdvanceSearch.js]: Any need of this commented code?

File: src/components/ColumnConfig.jsx [hunk 4]
```diff
@@ -24,13 +24,39 @@ class ColumnConfig extends Component{
...
     };
     this.setValuesOfVisibleColumnConfig = this.setValuesOfVisibleColumnConfig.bind(this);
+    this.setCheckValue = this.setCheckValue.bind(this);
   }
   setValuesOfVisibleColumnConfig (){
     this.props.setValuesOfVisibleColumnConfig(this.state.visibleColumnConfig);
     this.props.closeColumnOption();
   }
+  setCheckValue(){
+    let temp = this.state.selectValue = (this.state.selectValue === true)? false : true;
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: `temp` can have a better name

File: src/components/ColumnConfig.jsx [hunk 5]
```diff
@@ -24,13 +24,39 @@ class ColumnConfig extends Component{
...
+    this.setState({
+      selectValue: temp,
+    });
+    if(temp === true){
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]:
```suggestion
    if(temp){
```

File: src/components/ColumnConfig.jsx [hunk 6]
```diff
@@ -24,13 +24,39 @@ class ColumnConfig extends Component{
...
+      let temporaryVisibleColumnConfig = this.props.visibleColumnConfig;
+      Object.keys(temporaryVisibleColumnConfig).forEach((key) => {
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: Use the **for-in** loop instead

File: src/components/ColumnConfig.jsx [hunk 7]
```diff
@@ -24,13 +24,39 @@ class ColumnConfig extends Component{
...
+        temporaryVisibleColumnConfig[key]= true;
+      });
+      this.setState({
+        visibleColumnConfig: temporaryVisibleColumnConfig,
+      });
+    }
+    if(temp === false) {
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]:
```suggestion
    if(!temp) {
```

File: src/components/ColumnConfig.jsx [hunk 8]
```diff
@@ -24,13 +24,39 @@ class ColumnConfig extends Component{
...
+      let temporaryVisibleColumnConfig = this.props.visibleColumnConfig;
+      Object.keys(temporaryVisibleColumnConfig).forEach((key) => {
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: Use **for-in** loop instead.

File: src/components/ColumnConfig.jsx [hunk 9]
```diff
@@ -86,92 +112,100 @@ class ColumnConfig extends Component{
...
           </div>
         </form>
         <div className = "modal-save-container">
-          <div>
-            <button className="button-modal" onClick={() => this.checkAllColumns()}>Select All</button>
-            <button className="button-modal" onClick={() => this.unCheckAllColumns()}>Select None</button>
-          </div>
-          <div>
-            <button className="button-modal" onClick={this.props.closeColumnOption}>Close</button>
+          {/*<div className="select-button-wrapper">
```
Comment (pepcus-pratik) [inline src/components/ColumnConfig.jsx]: Commented code?

File: src/components/DataGrid.jsx [hunk 10]
```diff
@@ -221,16 +220,15 @@ class DataGrid1 extends Component {
...
-          <Redirect to={'/studentCorrection'}/>
-        </div> );
+      return <Redirect to={'/studentCorrection'}/>
     }
     return null;
   }
   EditButton = ({ rowData }) => (
-    <div className = "btn-block"><button onClick={() => { this.handleEditClick(rowData) }} className="btn-grid">
-      Edit</button>
+    <div className = "btn-block">
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]:
```suggestion
    <div className="btn-block">
```

File: src/components/DataGrid.jsx [hunk 11]
```diff
@@ -300,16 +298,17 @@ class DataGrid1 extends Component {
...
-              <button className="column-option-container" onClick={this.openColumnOption}>Column Options</button>
-              <ColumnConfig
-                columnOptionIsOpen= {this.state.columnOptionIsOpen}
-                closeColumnOption= {this.closeColumnOption}
-                visibleColumnConfig= {this.state.visibleColumnConfig}
-                setValuesOfVisibleColumnConfig = {this.setValuesOfVisibleColumnConfig}
-              />
-            </div>
+
             {/*<div>
```
Comment (pepcus-pratik) [inline src/components/DataGrid.jsx]: Commented Code??

File: src/components/SplashPage.jsx [hunk 12]
```diff
@@ -142,22 +142,22 @@ class SplashPage extends Component {
...
-          onClick={this._disableEditInfo}
-        />
       </div>
     )
   }
 
   renderAdminLoginFields() {
     return (
-      <div>
+      <div className="landingPageWrapper">
```
Comment (pepcus-pratik) [inline src/components/SplashPage.jsx]: incorrect practice for a class name

Comment (pepcus-pratik) [inline src/components/SplashPage.jsx]:
```suggestion
      <div className="landing-page-wrapper">
```

File: src/components/SplashPage.jsx [hunk 13]
```diff
@@ -174,16 +174,17 @@ class SplashPage extends Component {
...
-        />
-        <LinkButton
-          buttonText={loginBtnText}
-          linkPath={'/adminPanel'}
-          onClick={this._setAdminLogin}
-        />
-
+        <div className="adminButtonContainer">
+          <Button
+            buttonText={goBackBtnText}
```
Comment (pepcus-pratik) [inline src/components/SplashPage.jsx]: Avoid using the shorthand `btn` for `button`

File: src/tableGrid.css [hunk 14]
```diff
@@ -2,41 +2,42 @@
...
-    height:12px;
-    background-color: #fff;
-}
-.render-table::-webkit-scrollbar-thumb
-{
-    border-radius: 0px;
-    -webkit-box-shadow: inset 0 0 6px rgba(0,0,0,.3);
-    background-color: #e64d04;
-}
+/*.render-table::-webkit-scrollbar-track*/
```
Comment (pepcus-pratik) [inline src/tableGrid.css]: Commented Code??

File: src/components/AdvanceSearch.js (hunk 3 above)
Comment (BhagyashreeBhoyar) [reply src/components/AdvanceSearch.js]: I have Removed it.

File: src/components/ColumnConfig.jsx (hunk 4 above)
Comment (BhagyashreeBhoyar) [reply src/components/ColumnConfig.jsx]: I renamed temp to temporarySelecteValue.

File: src/components/AdminPanel.jsx (hunk 2 above)
Comment (priyalpepcus) [reply src/components/AdminPanel.jsx]: I asked Purvi and she told me that she will give me this text later.

File: src/components/SplashPage.jsx (hunk 13 above)
Comment (BhagyashreeBhoyar) [reply src/components/SplashPage.jsx]: It is already defined by Mohit.

File: src/components/SplashPage.jsx (hunk 12 above)
Comment (priyalpepcus) [reply src/components/SplashPage.jsx]: class name changed.

File: src/tableGrid.css (hunk 14 above)
Comment (priyalpepcus) [reply src/tableGrid.css]: done

**Pull request merged**